### PR TITLE
FIX Errors in the transformers integration docs

### PR DESCRIPTION
**Pull request by BenjaminBossan**
- Typo: wrong variable name
- Typo: `disable_adapter` -> `disable_adapters`
- Remove link to `PeftModel.disable_adapter`, because it's not the same method
- Mention the `enabble_adapters` method too

### CODE REVIEW

**Review by younesbelkada — APPROVED** [review]

File: docs/source/tutorial/peft_integrations.md [hunk 1]
```diff
@@ -141,10 +141,12 @@ output = model.generate(**inputs)
 print(tokenizer.decode(output_disabled[0], skip_special_tokens=True))
 ```
 
-To disable the adapter, call the [`~PeftModel.disable_adapter`] method.
+To disable the adapter, call the `disable_adapters` method.
```
Comment (stevhliu) [inline docs/source/tutorial/peft_integrations.md:144]: Is this the correct disable adapter method? It'd be nice to link to it if it is :)

```suggestion
To disable the adapter, call the [`~diffusers.loaders.PeftAdapterMixin.disable_adapters`] method.
```

File: docs/source/tutorial/peft_integrations.md [hunk 2]
```diff
@@ -141,10 +141,12 @@ output = model.generate(**inputs)
...
 
 ```py
-model.disable_adapter()
+model.disable_adapters()
 ```
 
+The `enable_adapters` can be used to enable the adapters again.
```
Comment (stevhliu) [inline docs/source/tutorial/peft_integrations.md:150]:
```suggestion
The [`~diffusers.loaders.PeftAdapterMixin.enable_adapters`] can be used to enable the adapters again.
```

**Review by stevhliu — APPROVED** [review]
Thanks for the fixes! I left a suggestion to update the correct methods with the proper links.

File: docs/source/tutorial/peft_integrations.md (hunk 2 above)
Comment (stevhliu) [reply docs/source/tutorial/peft_integrations.md:150]: Ah sorry, there are so many `enable_adapters` everywhere haha! Yeah, it should be [`~transformers.integrations.enable_adapters`] then, and the same for `disable_adapters`.

Comment (BenjaminBossan) [reply docs/source/tutorial/peft_integrations.md]: I tried:

```
[`~transformers.integrations.PeftAdapterMixin.enable_adapters`]
```

but the link is not added in the preview doc. Any idea what the issue might be?

Comment (BenjaminBossan) [reply docs/source/tutorial/peft_integrations.md]: Indeed there is no hit when I search for the methods in the transformers documentation. So what should I link to, a perma link to the source code?

Comment (stevhliu) [reply docs/source/tutorial/peft_integrations.md]: Yeah that sounds good to me!

**Pull request merged**